### PR TITLE
ci: updating nexus publish plugin version

### PR DESCRIPTION
**Pull request by tanderson-ld**
Updating nexus plugin

### CODE REVIEW

File: lib/shared/common/build.gradle.kts [hunk 1]
```diff
@@ -92,10 +88,7 @@ nexusStaging {
 nexusPublishing {
     clientTimeout.set(Duration.ofMinutes(2)) // we've seen extremely long delays in creating repositories
     repositories {
-        sonatype {
-            username.set(ossrhUsername)
-            password.set(ossrhPassword)
-        }
+        sonatype {}
```
Comment (kinyoklion) [inline lib/shared/common/build.gradle.kts:91]: I had to use findProperty to get the username/password here. Which allowed it to be undefined in development and populated for releases. Does some other magic make that not required?

Comment (tanderson-ld) [reply lib/shared/common/build.gradle.kts:91]: From my local testing, commands of this form work.

`./gradlew <task> -PsonatypeUsername=<username> -PsonatypePassword=<password>`

And this is what the publish script uses.

Comment (kinyoklion) [reply lib/shared/common/build.gradle.kts:91]: It also works when they are of form:
`./gradlew <task>`

File: lib/shared/internal/build.gradle.kts [hunk 2]
```diff
@@ -86,10 +82,7 @@ nexusStaging {
 nexusPublishing {
     clientTimeout.set(Duration.ofMinutes(2)) // we've seen extremely long delays in creating repositories
     repositories {
-        sonatype {
-            username.set(ossrhUsername)
-            password.set(ossrhPassword)
-        }
+        sonatype {}
```
Comment (kinyoklion) [inline lib/shared/internal/build.gradle.kts:85]: Parenthesis.

File: lib/shared/common/build.gradle.kts (hunk 1 above)
Comment (tanderson-ld) [reply lib/shared/common/build.gradle.kts]: Should be resolved based on slack discussion.

File: lib/shared/internal/build.gradle.kts (hunk 2 above)
Comment (tanderson-ld) [reply lib/shared/internal/build.gradle.kts]: Done

**Review by kinyoklion — CHANGES_REQUESTED** [review]
Realized some CI failed.

**Review by kinyoklion — APPROVED** [review]

**Review by kinyoklion — CHANGES_REQUESTED** [review]
Test packaging needs to disable signing?

**Review by kinyoklion — APPROVED** [review]

**Pull request merged**